themes/default/content/blog/pulumi-release-notes-69/index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "Pulumi Release Notes: Pulumi Import Improvements, RetainOnDelete as a resource option, and more!"
+allow_long_title: true
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2022-03-15T08:47:42-08:00
+
+# Use the meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. This field is required or the build will fail the linter test.
+meta_desc: The latest Pulumi updates also include the new `pulumi state rename` command, changing the default `pulumi plugin install` to the latest version, adding console output in non-interactive mode, and `pulumi cancel` support for self-managed state backends.
+
+# The meta_image appears in social-media previews and on the blog home page.
+# A placeholder image representing the recommended format, dimensions and aspect
+# ratio has been provided for you.
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - meagan-cojocar
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+---
+
+The team has been busy releasing new features and improvements in the last 3 weeks. Read on to learn about what's new in this release!
+
+- Pulumi CLI and core technologies
+  - [`pulumi import` Improvements](#pulumi-import-improvements)
+  - [Add `RetainOnDelete` as a resource option](#add-retainondelete-as-a-resource-option)
+  - [New `pulumi state rename` command](#new-pulumi-state-rename-command)
+  - [Default `pulumi plugin install` to the latest version](#default-pulumi-plugin-install-to-the-latest-version)
+  - [Add console output in non-interactive mode](#add-console-output-in-non-interactive-mode)
+  - [Support `pulumi cancel` for filestate backends](#support-pulumi-cancel-for-filestate-backends)
+
+<!--more-->
+
+
+## Pulumi CLI and core technologies
+
+### `pulumi import` Improvements
+
+Last year, we introduced a new Pulumi import feature that allows you to import existing infrastructure into your Pulumi program. We’ve listened to feedback and delivered a plethora of updates and fixes to streamline the import experience; to make it more useful, more convenient, and more powerful. Not only can this feature bring a resource into your Pulumi state file, but it can also generate the source code for your Pulumi program too. 
```

Comment:
Super tiny nit, no need for a semicolon here (though a comma would be okay if you wanted):

```suggestion
Last year, we introduced a new Pulumi import feature that allows you to import existing infrastructure into your Pulumi program. We’ve listened to feedback and delivered a plethora of updates and fixes to streamline the import experience to make it more useful, more convenient, and more powerful. Not only can this feature bring a resource into your Pulumi state file, but it can also generate the source code for your Pulumi program too. 
```